### ISSUE TITLE
Writing an empty numpy array into an empty RingBuffer "fills up" the buffer

### ISSUE THREAD

**Issue by egorchakov** [justengel/np_rw_buffer#1]
A minimal example:

```
In [1]: import np_rw_buffer, numpy as np                                                                               

In [2]: rb = np_rw_buffer.RingBuffer(shape=(1234,1))                                                                   

In [3]: len(rb)                                                                                                        
Out[3]: 0

In [4]: rb.write(np.array([]))                                                                                         

In [5]: len(rb) # <---                                                                                                        
Out[5]: 1234

In [6]: rb.write(np.array([1,2,3,4]))                                                                                  
<...>
OverflowError: Not enough space in the buffer <np_rw_buffer.buffer.RingBuffer object at 0x7fe2340da748> 1234 < 4
```

**Comment by justengel** [comment]
Thanks! It should be fixed in version 1.1.7

**Comment by egorchakov** [comment]
Thank you!
